meshers/blocky/voxel.cpp

```diff
@@ -253,6 +299,7 @@ static void bake_mesh_geometry(Voxel &config, Voxel::BakedData &baked_data) {
 	PoolVector3Array positions = arrays[Mesh::ARRAY_VERTEX];
 	PoolVector3Array normals = arrays[Mesh::ARRAY_NORMAL];
 	PoolVector2Array uvs = arrays[Mesh::ARRAY_TEX_UV];
+	PoolVector<real_t> tangents = arrays[Mesh::ARRAY_TANGENT];
```

Comment:
`real_t` could be `double`, and this is way too much for a tangent. I think it should be made `float` explicitely.

Reply:
Godot's visual server expects a real array, so I made it one worried I might break it. I'll give this a go.